<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TIA</class>
 <widget class="QMainWindow" name="TIA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TIA</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDebugger">
    <property name="title">
     <string>Debugger</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebugger"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionReload_File">
   <property name="text">
    <string>Reload File</string>
   </property>
  </action>
  <action name="actionReload_DB">
   <property name="text">
    <string>Reload DB</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
